source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,20 @@
 ============
 Introduction
 ============
+
+oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
+ from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.
+
```

Comment:
Here we can add the paragraph related to the concept of end-to-end and device agnostic interfaces
from https://software.intel.com/en-us/daal-programming-guide-introducing-the-intel-data-analytics-acceleration-library
Does it make sense to have visual representation of the things probably in the different format? 
Is this feasible?

Reply:
I've add section. as for visual representation - we can use existing images as there is no centralized design/visualization guidelines in spec and we can rewrite this later

Reply:
let's use them, if possible

Reply:
done